### PR TITLE
Cloud reference overhaul

### PR DESCRIPTION
**Pull request by matkuliak**
## Summary of the changes / Why this is an improvement

Updated reference with the latest releases
Also reorganized it quite a bit to better fit the current UI

## Checklist

 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

**Comment by matkuliak** [comment]
This is a pretty big reorg. so it would probably be easier to review it by deploying it locally. Feel free to message me if there's any trouble with that 🙏🏻

File: docs/overview.rst [hunk 1]
```diff
@@ -60,50 +59,48 @@ Once signed in, you will be presented with the Organization overview.
 Organization
 ============
 
-The Organization overview lists the organizations you can access.
-For the first-time users, an organization called "My organization" is 
-automatically crated upon first login.
+The organization is the highest structure in yout CrateDB Cloud Console.
```
Comment (SStorm) [inline docs/overview.rst:62]:
```suggestion
The organization is the highest structure in your CrateDB Cloud Console.
```

File: docs/overview.rst [hunk 2]
```diff
@@ -60,50 +59,48 @@ Once signed in, you will be presented with the Organization overview.
...
+Multiple clusters and users can exist in a organization at any moment. For the
```
Comment (SStorm) [inline docs/overview.rst:63]:
```suggestion
Multiple clusters and users can exist in a organization at any moment. For 
```

File: docs/overview.rst [hunk 3]
```diff
@@ -60,50 +59,48 @@ Once signed in, you will be presented with the Organization overview.
...
+first-time users, an organization called "My organization" is automatically 
+crated upon first login.
```
Comment (SStorm) [inline docs/overview.rst:65]:
```suggestion
created upon first login.
```

File: docs/overview.rst [hunk 4]
```diff
@@ -60,50 +59,48 @@ Once signed in, you will be presented with the Organization overview.
...
    :alt: Cloud Console organization overview
 
-The Organization overview consists of seven tabs: *Dashboard*, *Settings*,
-*Users*, the *Audit Log*, *Billing*, *Payment Methods*, and finally,
-*Regions*. By default it shows the Dashboard tab, which displays two things.
-On the left, it shows a link for creating a new cluster, the blue *Deploy new
-cluster* button. On the right, you will see any already deployed clusters.
+The Organization overview consists of seven tabs: *Clusters*, *Settings*,
+*Users*, *Billing*, *Payment Methods*, *Audit Logs*, and *Regions*. By default
+you are brought to the Clusters tab, which provides quick overview of all
```
Comment (SStorm) [inline docs/overview.rst:75]:
```suggestion
you are brought to the Clusters tab, which provides a quick overview of all
```

File: docs/overview.rst [hunk 5]
```diff
@@ -128,28 +125,14 @@ your organization, they will receive an email invitation to do so.
...
 .. _overview-org-billing:
 
 Organization Billing
 --------------------
 
-This tab shows all your existing subscriptions, along with which cluster is
-currently using the subscription. Current accumulated billing snapshot is also
-visible here, along with additional information:
+The Billing tab shows all your existing subscriptions, along with which
+cluster is currently using the subscription. Current accumulated billing
```
Comment (SStorm) [inline docs/overview.rst:134]:
```suggestion
cluster is currently using the subscription. The current accumulated billing
```

File: docs/overview.rst [hunk 6]
```diff
@@ -174,6 +159,19 @@ Cloud subscription payment methods can also be added here.
 .. image:: _assets/img/payment-methods2.png
    :alt: Cloud Console payment methods
 
+.. _overview-org-audit:
+
+Organization Audit Logs
+-----------------------
+
+This tab shows the Audit Logs of current organization.
```
Comment (SStorm) [inline docs/overview.rst:167]:
```suggestion
This tab shows the Audit Logs of the current organization.
```

File: docs/overview.rst [hunk 7]
```diff
@@ -255,58 +258,96 @@ information about the cluster. This includes:
...
-* **Date created**: The day of the original deployment of the cluster.
+* **Connect to your cluster**: Click the "view code snippets" link to
+  see a list of snippets for the available clients and libraries. These
+  include: CLI clients, Java, JavaScript, PHP, Python, and Ruby.
 
-* **DB Username**: This shows once more the username associated with the
-  cluster. You chose this username while setting up the cluster originally.
+* **Demo data**: Clicking the "import the demo data" will show you two
+  snippets that can be used in the SQL Console of your Admin UI. The first
+  snippet a "nyc_taxi" table, and the second one will populate it with our
```
Comment (SStorm) [inline docs/overview.rst:272]:
```suggestion
  snippet will create a "nyc_taxi" table, and the second one will populate it with our 
```

File: docs/overview.rst [hunk 8]
```diff
@@ -255,58 +258,96 @@ information about the cluster. This includes:
...
-On the right you will see more immediate metrics, collected from the last hour
-of cluster operations. This includes average queries, average query response
-time, and disk, memory, and CPU usage for the last hour.
+Credentials
+~~~~~~~~~~~
 
-The queries and query response time of the cluster can be seen in more detail
-on the Cluster Metrics page, for which see below. This page can also be
-accessed by clicking the *View metrics* link at the top right.
+Here you can change your CrateDB Console password.
```
Comment (SStorm) [inline docs/overview.rst:298]:
```suggestion
Here you can change your CrateDB database password.
```

File: docs/overview.rst [hunk 9]
```diff
@@ -255,58 +258,96 @@ information about the cluster. This includes:
...
 
+.. _overview-cluster-overview-security:
+
+Security
+~~~~~~~~
+
+Click the box labelled *Deletion Protected* to add deletion protection to your
+cluster. This will ensure that it can only be deleted in either the CrateDB
+Console or croud by a user with :ref:`org admin rights <org-roles>` (or by a
+Crate.io employee superuser).
```
Comment (SStorm) [inline docs/overview.rst:308]: This is not the case - deletion protection ensures that it cannot be deleted by anyone, unless deletion protection is switched off. The intent here is to make deletion a doubly-conscious choice and not accidentally delete your production cluster.

File: docs/overview.rst [hunk 10]
```diff
@@ -255,58 +258,96 @@ information about the cluster. This includes:
...
+allowlist.
+
+.. _overview-cluster-overview-private-endpoints:
+
+Private endpoints
+~~~~~~~~~~~~~~~~~
+
+A private endpoint, or private link, is a mechanism that allows a secure, 
+private connection to your cluster. Effectively, it allows you to bypass the
+public internet when accessing the environment where your cluster is deployed.
```
Comment (SStorm) [inline docs/overview.rst:334]: Maybe worth mentioning that it cannot be used across providers - i.e. from your Azure account to your AWS cluster.

File: docs/overview.rst [hunk 11]
```diff
@@ -405,65 +444,35 @@ The contributions of each type of query to the total results displayed in the
...
 
-Cluster Scale
-'''''''''''''
+Now you can do three different things:
 
-The blue box on the left shows the current cluster subscription plan. The
-scale unit can be adjusted on this page, and the storage capacity can be
-increased here.
+- Change the plan of your cluster
+- Increase node's storage
```
Comment (SStorm) [inline docs/overview.rst:468]:
```suggestion
- Increase storage on each node
```

File: docs/overview.rst [hunk 12]
```diff
@@ -405,65 +444,35 @@ The contributions of each type of query to the total results displayed in the
...
+- Icrease/decrease the number of nodes
 
-.. image:: _assets/img/cluster-settings-scale.png
-   :alt: Cloud Console cluster scaling tab
+You can do only one of those operations each time, i.e. you can't change plane
```
Comment (SStorm) [inline docs/overview.rst:471]:
```suggestion
You can do only one of those operations at a time, i.e. you can't change plans
```

File: docs/overview.rst [hunk 13]
```diff
@@ -473,58 +482,32 @@ panel. The cluster capacity indicated will change accordingly.
...
 and, if a :ref:`minor upgrade <gloss-version>` or
 :ref:`patch upgrade <gloss-version>` is possible, a list of available
 upgrades. If no upgrade is possible, because the cluster is up to date with 
 the latest version of CrateDB, nothing will be displayed in the Available
-Upgrades panel. You can upgrade a cluster from the Available Upgrades panel by
-clicking the *Upgrade* button on the right. Upgrading may take a brief moment,
-during which other cluster operations will be disabled.
+Upgrades panel. You can upgrade a cluster from the Available upgrades panel by
+clicking the *Upgrade* button next to a specific shown version. Upgrading may
+take a brief moment, during which other cluster operations will be disabled.
```
Comment (SStorm) [inline docs/overview.rst:504]:
```suggestion
clicking the *Upgrade* button next to a specific shown version. Depending on how much data you have in your cluster, upgrading might take a while, during which other cluster operations will be disabled. Your cluster's nodes
will be upgraded one at a time, so your cluster will remain available through the process.
```

**Review by SStorm — COMMENTED** [review]
Thank you for bring this up to date. I've added a few comments for typos and some other suggestions for some things that are not entirely accurate :)

**Review by kojinkai — APPROVED** [review]

**Pull request merged**